frontend/src/components/Course.tsx

```diff
@@ -53,22 +32,16 @@ function Course(props: any) {
         </Typography>
```

Comment:
Can you complete the refactor relating to modals, by removing the description from this card component, and possibly also removing the Pre-req, restriction, dependent info?